### PR TITLE
docs: re-organization

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
This PR restructures the information layout in the main menu (navbar).  

💻 [Preview Link](https://deploy-preview-1009--docs-spectrocloud.netlify.app/)

CleanShot 2022-11-29 at 16 35 46

## What is Palette?

This section was previously titled "Introduction to Spectro Cloud." We are better off explaining what Palette is and its use cases.  Previously, an architecture document resided in this section but is now moved to the Architecture section.  Why Spectro Cloud is handled by the main marketing page, the visitors of the documentation page are current and new users of Palette, therefore, the focus should be on Palette vs. the organization as a whole.

## Getting Started

This section is removed until the first tutorial is released. This section is intended to help the user get started with the product. Today, this section does not fulfill that purpose and primarily contains irrelevant information on getting started with the product. The discussion about the freemium model is better left off in the What is Palette section.

The Getting Started needs to help the user understand App Mode's journey and Cluster mode's journey. 

## Architecture

A section dedicated to the architecture and technical details of the product is missing (excluding security). This section has been added along with elements from the troubleshooting section that better fit an architecture topic. 

CleanShot 2022-12-01 at 09 57 25

## Troubleshooting
As mentioned in the Architecture section, this section was cleaned up and now only focuses on troubleshooting steps.  The existing content has been broken up into their respective category and file..

CleanShot 2022-12-01 at 09 56 18

The content of the index page.
CleanShot 2022-12-01 at 10 01 38

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1009#" title="Author self-approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
@karl-cardenas-coding: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | d2841e5cb860acb8e935e2f69f2bef5f381270ce | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1009/librarium_build/1598359434688466944) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by antongisli** [comment]
LGTM with one caveat - while removing the getting started for now, what will users see first? what will their journey be like, will they get even more lost?

also - the explanation about cluster mode vs app mode needs to be early & crisp, it has a large potential to confuse different personas if we get it wrong.

**Comment by karl-cardenas-coding** [comment]
@antongisli I agree, but until we have the first tutorial, I think it's best to omit it for now as it's not really helping the user. But I can understand if we want to keep it but we wouldn't have any content for it so it's kinda awkward to have an empty section.  

We should create an App Mode vs Cluster Mode page and add it in the What is Palette? to people understand the two concepts early.

File: content/docs/14-troubleshooting.md
Comment (ritawatson) [inline content/docs/14-troubleshooting.md:18]:
```suggestion
Use the following troubleshooting resources to help you address issues that may arise. You can also reach out to our support team by opening up a ticket through our [support page](http://support.spectrocloud.io/).
```

**Review by ritawatson — APPROVED** [review]
LGTM. Very good restructure.

A couple comments:
- 'Hence reduces...' could be updated to say 'This reduces...'
- Do we want to change 'On-Premises Installation' to 'On-Prem Deployment'?

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1009#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1009#pullrequestreview-1203045821" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by resmaa — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1009#" title="Author self-approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by karl-cardenas-coding** [comment]
Outdated and no longer needed.

**Pull request closed without merging**
